webpack.config.js

```diff
@@ -75,6 +79,15 @@ module.exports = {
                 // Use a content-based hash in the name so that we can set a long cache
                 // lifetime for assets while still delivering changes quickly.
                 oneOf: [
+                    {
+                        // allow runtime loading of twemoji (at the expense of cachebusting).
+                        resource: /TwemojiMozilla.*\.woff2$/,
```

Comment:
It should be possible to load these fonts in the font manager via `require` as we do for images in JS, which would allow us to keep the cache busting, and we wouldn't need any changes here.